textio.yml

```diff
@@ -0,0 +1 @@
+team: product-services
```

Comment:
❓ Does textio.yml not need a `name:` key? All the ones I've seen have the project name as well, like [this](https://github.com/textioHQ/docparty-backend/blob/main/textio.yml)

Reply:
I am not sure. What @oliviagunton said is this:
```I think if you add textio.yml with no name (since we don’t want Circle build) but team: product-services (or one of the other legacy team names) that should give our whole team admin access.```

Reply:
But I actually think having a circle build is what I am going for, ultimately, so that circle can do the deploy to pypi, so I will add a name.

Reply:
Yeah, `name` is not needed until you want circle builds. If you add it, every commit will look like the build is failing (as in https://github.com/textioHQ/infrastructure-team-admins) so I strongly (⭐ ⭐ ) encourage not adding it until you're ready to also add a circle config.